src/checkstyle/Checker.hx

```diff
@@ -134,10 +137,15 @@ class Checker {
 			var flagValue = define.split("=");
 			parser.define(flagValue[0], flagValue[1]);
 		}
-		return parser.parse();
+
+		try {
+			return parser.parse();
+		}
+		catch (e:Dynamic) {}
```

Comment:
I don't like this, this creates another "swallowed errors" problem (see #154). It makes sense to continue, but we should still output the error.